pkg/agent/datamodel/types.go

```diff
@@ -1261,6 +1262,23 @@ func (k *KubernetesConfig) IsIPMasqAgentEnabled() bool {
 	return k.IsAddonEnabled(IPMASQAgentAddonName)
 }
 
+func (k *KubernetesConfig) IsDualStack() bool {
```

Comment:
is it possible to set `EnableIPv6DualStack` in `FeatureFlags` when DualStack is enabled so we do not change these code in AgentBaker?

Reply:
I avoided using the FeatureFlags because we do not currently use it for Kubenet dual stack so if we start using it here then it could have unintended consequences on the Linux side.

Reply:
How does Linux support IPv6? should we also need to update other components if we enable IPv6 on Windows agent pools, for example, LoadBalancerSku?

Reply:
everything on the Linux side is already taken care of, we just don't happen to use the "FeatureFlag" property for it. It's controlled via toggle and then a cluster can be checked if it is dualstack by looking at the ipFamilies property on the managed cluster to see if it is `ipv4` or `ipv4,ipv6`.

Reply:
@tyler-lloyd you can search `EnableIPv6DualStack` in AKS RP code and check whether it is needed when using ipFamilies property